src/main/java/no/nav/pus_fss_frontend/config/ApplicationConfig.java

```diff
@@ -49,4 +53,19 @@ public SelfTestMeterBinder selfTestMeterBinder(SelfTestChecks selfTestChecks) {
         return new SelfTestMeterBinder(selfTestChecks);
     }
 
+    @Bean
+    public WebServerFactoryCustomizer<ConfigurableServletWebServerFactory> webServerFactoryCustomizer(EnvironmentProperties environmentProperties) {
+        return (factory) -> {
+            String contextPath = getContextPath(environmentProperties);
+            factory.setContextPath(contextPath);
+        };
+    }
+
+    static String getContextPath(EnvironmentProperties environmentProperties) {
+        String appContextPathEnv = environmentProperties.getContextPath();
```

Comment:
Kunne vurdert å bruke NULL som default istedenfor en tom streng for `naisAppName` og  `contextPath`

Reply:
@AlexGaard 
Om man har 
```app.env.contextPath=${APP_CONTEXT_PATH}```
 i properties og environment-variabelen ikke er satt så blir propertien satt til strengen "${APP_CONTEXT_PATH}"

Løser derfor optional med å sette den slik: 
```app.env.contextPath=${APP_CONTEXT_PATH:}```
Om environment-variabelen da ikke er satt så blir propertien satt til en tom streng "". 

Hva tenkte du med NULL? 
```app.env.contextPath=${APP_CONTEXT_PATH:NULL}```   
==> propertien er strengen "NULL"

Reply:
Aha, tror jeg misforsto. Tenker du at getContextPath() burde returnere NULL?

Det kan den ikke fordi 
```factory.setContextPath(contextPath);```
Feiler om contextPath = null. Da gir den beskjed om at default er tom streng :)

Reply:
```app.env.contextPath=${APP_CONTEXT_PATH} ```
Fungerer dessverre dårlig når APP_CONTEXT_PATH ikke er satt. 

```Tomcat started on port(s): 8080 (http) with context path '/${APP_CONTEXT_PATH}'```

Reply:
Så kunne man brukt Optional til
```
        return ofNullable(environmentProperties.getContextPath())
                .or(() -> ofNullable(environmentProperties.getNaisAppName()))
                .map((contextPath) -> Utils.ensureStringIsPrefixed(contextPath, "/"))
                .orElse("");
```

Reply:
Men det er bare et forslag, PRen ser bra ut :)

Reply:
Jeg synes det var et veldig godt forslag og det ser veldig fint ut. 
Men virker som det er vanskelig å få environmentProperties.getContextPath() til å returnere null. 
Uten å gjøre endringer på selve getContextPath() funksjonen :)

Reply:
Nei enig! 😂 
Rar oppførsel av spring/.properties her